src/utils/dev.ts

```diff
@@ -207,7 +207,7 @@ class NuxtDevServer extends EventEmitter {
     }
 
     // Remove websocket handlers on close
-    this._currentNuxt.hooks.hookOnce('close', () => {
+    this._currentNuxt.hooks.hook('close', () => {
       this.listener.server.removeAllListeners('upgrade')
```

Comment:
```suggestion
    const unsubClose = this._currentNuxt.hooks.hook('close', () => {
      unsubClose()
      this.listener.server.removeAllListeners('upgrade')
```

Reply:
Is there a chance we might polyfill hookOnce from once place? Ideally in core or bridge but since i support requires more work, in kit loader at least to patch and support `hookOnce`. Making sure bridge keeps working in long term via this patches seems less practical to me.

Reply:
That would be better, for sure.

Reply:
Separate topic how long we should support nuxt 2 / bridge for nuxi we can chat later -- specially as first class (which I'm not sure is also being tested against nuxt/nuxt/kit)

In the meantime yeah let's find a better way to make sure at least regressions with hookOnce won't happen in a safer way.

Reply:
we do test bridge against nuxt with ecosystem-ci before every release of nuxt